File: week-2/fibonacci-iterative/fibonacci-iterative.cpp
#include <iostream>
using namespace std;

int main(){
	int number = 0, first = 0, second = 1, tmp = 0;
	cout<<"Enter number "<<endl;
	cin >> number;
	for(int i=0; i<number;i++){
		tmp = first + second;
		first = second;
		second = tmp;
		
	}
	cout<<first<<endl;
	return 0;
}

File: week-2/fibonacci-iterative/fibonacci-iterative.py
num = int(input("Enter Number ").strip())

arr = []
def fibonacci(num):
    first = 0
    second = 1
    sum = 0
    for i in range(num):
        arr.append(first)
        first, second = second , first + second
    return first
print(fibonacci(num))
print("list of fib.. ",arr)
